.github/workflows/build_linux_packages.yml

```diff
@@ -136,14 +142,30 @@ jobs:
 
       # TODO: Move to script
       - name: Upload Logs
+        if: always()
         run: |
-          aws s3 cp build/logs/ s3://therock-artifacts/${{github.run_id}}/logs/${{env.AMDGPU_FAMILIES}}/ \
-            --recursive \
-            --content-type="text/plain" \
-            --exclude "*" \
-            --include "*.log"
-          aws s3 cp build/logs/index.html s3://therock-artifacts/${{github.run_id}}/logs/${{env.AMDGPU_FAMILIES}}/
-
+          if [ ! -d build/logs ]; then
+            echo "[INFO] Logs directory missing. Creating build/logs..."
+            mkdir -p build/logs
+          fi
+
+          echo "[INFO] Uploading logs..."
+
+          if compgen -G "build/logs/*.log" > /dev/null; then
```

Comment:
This `compgen` command is not portable to Windows. Switch to Python.

Reply:
Sounds good!